AmplifyPlugins/API/AWSAPICategoryPluginTests/AWSAPICategoryPluginTestBase.swift

```diff
@@ -62,7 +63,6 @@ class AWSAPICategoryPluginTestBase: XCTestCase {
             XCTFail("Failed to create endpoint config")
         }
 
-        Amplify.reset()
```

Comment:
This reset, above configuring the APIPlugin, was causing the URLSession error `Task created in a session that has been invalidated`. I didn't look too deep into this, but i'm assuming it is due to the URLSession being created, invalidated, and used.